pkg/client/docker/daemon.go

```diff
@@ -378,8 +378,10 @@ func LaunchDaemon(ctx context.Context, name string) (conn *grpc.ClientConn, err
 	case cid := <-cidFound: // Success, the daemon info file exists
 		err := cache.SaveDaemonInfo(ctx,
 			&cache.DaemonInfo{
-				Options:  map[string]string{"cid": cid},
-				InDocker: true,
+				Options:       map[string]string{"cid": cid},
+				InDocker:      true,
+				DaemonAddress: addr.String(),
```

Comment:
I'd prefer if we just add the port here. The host differs depending on if you're on the host (127.0.0.1) or if you're in a container (DNS name of daemon).

Reply:
understood; @njayp does that sound good to you? we should still be able to dial using only the port, yea?

Reply:
the kubecontext will still be here, it's just a question of whether the address includes the host or just the port

Reply:
ah ok, sound good to me, ty